app/models/item.rb

```diff
@@ -7,4 +7,10 @@ class Item < ActiveRecord::Base
   belongs_to :category
   has_many :order_items
   has_many :orders, through: :order_items
+
+  def self.search(query)
+    where('LOWER(brand) ILIKE ?', "%#{query}%") |
+      where('LOWER(name) ILIKE ?', "%#{query}%") |
+      where('LOWER(description) ILIKE ?', "%#{query}%")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.